keycloak-apikey/src/main/java/eu/europeana/keycloak/validation/service/SesstionTrackerUpdator.java

```diff
@@ -0,0 +1,92 @@
+package eu.europeana.keycloak.validation.service;
+
+import eu.europeana.keycloak.KeycloakUtils;
+import eu.europeana.keycloak.validation.datamodel.ErrorMessage;
+import eu.europeana.keycloak.validation.datamodel.SessionTracker;
+import eu.europeana.keycloak.validation.util.Constants;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.BiFunction;
+
+/**
+ *
+ */
+public class SesstionTrackerUpdator implements BiFunction<String, SessionTracker, SessionTracker>,
```

Comment:
I propose to rename this class to SessionTrackerUpdater, so we can more easily find back the class.

Reply:
Also, could you please add a short description (javadoc)